sdcm/tester.py

```diff
@@ -1620,7 +1620,7 @@ def run_stress_cassandra_thread(
                                           profile=profile,
                                           node_list=self.db_cluster.nodes,
                                           round_robin=round_robin,
-                                          client_encrypt=self.db_cluster.nodes[0].is_client_encrypt,
+                                          client_encrypt=self.params.get('client_encrypt'),
```

Comment:
Is this set in the longevity yaml now as false?
If so, we need to set there a comment to change it to true once we support it.
If not, please add it.

Reply:
Both on sct and siren-test's defaults it's set to false, and in siren test there's already a comment https://github.com/scylladb/siren-tests/blob/master/sct_plugin/configurations/scylla_cloud_defaults.yaml#L9

Reply:
I want it specifically in the relevant yaml, even if the defaults are false.
There it should be mentioned that this is a workaround and need to be set to "true" once the client_encryption is supported for cloud.

Reply:
This is the yaml of the run in question https://github.com/scylladb/siren-tests/blob/master/sct_plugin/configurations/scylla_cloud_nemesis_small_set.yaml#L30
The comment exists there as well